serverless/python-template/StrategyHandler.py

```diff
@@ -110,6 +139,10 @@ def get_indicator(self, indicator_name):
             logging.info(response)
         return self.indicators[indicator_name]
 
+    #############################################################
+    ####### I don't think we need any of the functions below
+    ####### Or at least I don't see how they would be usable in an SNS-triggered architecture
+    ##############################################################
```

Comment:
cool. you can remove them. I copied them in as a reminder to write what's needed for querying the database. maybe better to leave a # todo and refer to this code in core for reference on building the query stuff